toast-tk-runtime/src/main/java/io/toast/tk/runtime/AbstractTestPlanRunner.java

```diff
@@ -188,19 +182,6 @@ private void execute(final ITestPlan testPlan, final TestRunner runner) {
 		tearDownEnvironment();
 	}
 
-	/**
-	 * Checks if report must be sent by mail. See "mail.send" in toast.properties.
-	 *
-	 * @return True if the report must be sent by mail.
-	 */
-	private boolean shouldSendMail() {
-		return getConfig().isMailSendReport();
-	}
-
-	private AdaptersConfig getConfig() {
-		return new AdaptersConfigProvider().get();
-	}
-
 	private void createAndOpenReport(final ITestPlan testPlan) {
 		final String path = getReportsFolderPath();
 		final String pageName = "testplan_report";
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)